README.md

```diff
@@ -92,12 +92,37 @@ the [role_binding.yaml](deploy/role_binding.yaml).
 
 ### Building the Operator Image
 
-To build the operator,
+There are two options to build the operator image. You can choose either `operator-sdk` or `ko`.
+
+#### To build the operator with `operator-sdk`:
 
 ```
 operator-sdk build quay.io/$REPO/knative-serving-operator:$VERSION
```

Comment:
Do we want to outline which method users should prefer? I am of the mindset that since everything in Knative is using ko, that would be the recommended/supported path. And it would also be how the official Knative releases will be done.

Reply:
I concur. There's no reason to build with operator-sdk if the test-infra is doing it.

Reply:
OK, I will remove the sdk part.